RS: Everest release notes draft (#2718)
RedisLabs/redislabs-docs: content/rs/release-notes/rs-7-2-releases/rs-7-2-tba.md

- Shard IDs in cluster shards topology

- Introduce shard ID to Redis cluster

- Support `CLIENT NO-TOUCH` command

- `WAIT AOF`

#### Three Redis database versions

Redis Enterprise Software version 6.x includes two Redis database versions: 6.0 and 6.2. As of version 7.2, Redis Enterprise Software includes three Redis database versions: 6.0, 6.2, and 7.2.

To view available Redis database versions:

- In the Cluster Manager UI, see **Redis database versions** on the **Cluster > Configuration** screen.

- Send a [`GET /nodes` REST API request]({{<relref "/rs/references/rest-api/requests/nodes">}}) and see `supported_database_versions` in the response.

The default Redis database version, which is used when you upgrade an existing database or create a new one, differs between Redis Enterprise releases as follows:

| Redis<br />Enterprise | Bundled Redis<br />DB versions | Default DB version<br />(upgraded/new databases) |
|-------|----------|-----|
| 7.2 | 6.0, 6.2, 7.2 | 7.2 |
| 6.4.2 | 6.0, 6.2 | 6.2 |
| 6.2.x | 6.0, 6.2 | 6.0 |

For Redis Enterprise Software version 7.2, `default_redis_version` is 7.2 for both `major` and `latest` upgrade policies.

> #### Auto Tiering - Redis on Flash evolution doubling performance {#auto-tiering}

Reviewer: ```suggestion
#### Auto Tiering - Redis on Flash evolution doubling throughput while cutting latencies in half {#auto-tiering}
```

> Redis Enterprise version 7.2 introduces Auto Tiering as an enhanced successor to Redis on Flash, which allows you to provision larger databases at a lower cost by using NVMe drives in addition to RAM.

Reviewer: ```suggestion
Redis Enterprise version 7.2 introduces Auto Tiering as an enhanced successor to Redis on Flash, which allows you to provision larger databases at a lower cost by extending the RAM with flash drives.
```

> Redis Enterprise Auto Tiering uses [Speedb](https://www.speedb.io/) as its storage engine, doubling the throughput achieved. For example, a 1 TB database with 50K ops/sec can now serve 100K ops/sec based on the same infrastructure.

Reviewer: ```suggestion
Redis Enterprise Auto Tiering replaces RocksDB with [Speedb](https://www.speedb.io/) as its storage engine, doubling the throughput and cutting latencies, achieved using the same infrastructure resources. For example, a 1 TB database with 50K ops/sec can now serve 100K ops/sec based on the same infrastructure.
```

For more information about Auto Tiering, see:

- [Auto Tiering overview]({{<relref "/rs/databases/auto-tiering">}})

- [Auto Tiering quick start]({{<relref "/rs/databases/auto-tiering/quickstart">}})

> 

Reviewer: I would like to include here another sentence or paragraph that explains what customers need to do in order to utilize Auto Tiering with Speedb:
Customers will need to upgrade their cluster to RS 7.2, and then upgrade each of their Auto Tiering enabled databases to v7.2 (DB version) so it will start using Speedb and get the performance improvements.

Author: Added this info

#### Updated Redis Enterprise license format

Redis Enterprise Software version 7.2 includes updates to its license format, which add separate shard limits for RAM and flash shards used for Auto Tiering.

> 

Reviewer: I would like to add a "read more" with a link to:
/rs/clusters/configure/license-keys/

Author: Added this link

- Provides configuration flexibility while highlighting the recommended path

- Addresses the needs of different personas and use cases

- Quicker troubleshooting and easier maintenance

##### New UI highlights

- More configurable database attributes, including replica high availability, shards placement, and proxy policy.

- Nodes indicate whether it’s a primary or secondary node.

- Modules show the databases that are using them.

- Certificates show expiration and validity, and you can upload and copy certificates.

- Cluster license displays the number of shards that are used out of the number of shards that are licensed to the cluster. The new UI allows you to paste or upload a new license.

- Role-based access control (RBAC) has explanations to improve clarity. 

- Access Control List (ACLs) now support defining ACLs for modules.

- The databases screen has more information per database for faster troubleshooting. It also allows you to filter databases and compare database metrics.

- The cluster name, user, and user role are shown in the upper right for quickly identifying the cluster from any screen. You can also **Change user password** from the dropdown menu.

> - Auto Tiering licensing and an emergency switch for the flash engine (available only in the new UI).

Reviewer: I don't like this "emergency switch". It makes it feel like we are not confident in the new feature. The UI and other documentation pieces have enough warnings to reflect when to use and how.
```suggestion
- Auto Tiering licensing and a toggle for the storage engine used in Auto Tiering enabled databases (available only in the new UI).
```
